src/spaceone/monitoring/manager/monitoring_manager.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from spaceone.core.manager import BaseManager
+from spaceone.core import utils
+from spaceone.core.utils import get_dict_value
+from spaceone.monitoring.conf.monitoring_conf import *
+from spaceone.monitoring.connector.kubernetes_connector.pod_log import PodLog
+from spaceone.monitoring.model.log_model import Log, PodLogInfo
+import dateutil.parser as date_parser
+
+_LOGGER = logging.getLogger(__name__)
+
+
+
+class MonitoringManager(BaseManager):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def list_logs(self, params):
+        results = []
+        kubernetes_log_connector: PodLog = self.locator.get_connector('PodLog', **params)
+        kubernetes_log_connector.set_connect(params.get('options'), params.get('secret_data'))
+
+        logs = kubernetes_log_connector.list_logs(params)
+
+        for i in range(len(logs)):
```

Comment:
Please change the `i` to mean the index count of the log to be cleaner.

Reply:
I see. For clarity I agree with this comment and changed it accordingly :)